packages/doenetml-worker-rust/lib-doenetml-core/src/core/dependency.rs

```diff
@@ -87,10 +87,10 @@ impl TryFrom<&DependencySource> for StateVarPointer {
 #[derive(Debug)]
 pub struct Dependency {
     pub source: DependencySource,
-    pub value: StateVarReadOnlyViewEnum,
+    pub value: StateVarViewEnum,
 }
 
-/// The vector of dependencies that were created for a `DependencyInstruction`
+/// The vector of dependencies that were created for a `DataQuery`
 #[derive(Debug)]
 pub struct DependenciesCreatedForInstruction(pub Vec<Dependency>);
```

Comment:
This should become something like `DependenciesCreatedForQuery`.

Reply:
Done